includes/optional-modules/class-rss.php

```diff
@@ -339,6 +373,22 @@ public static function render_content_settings_metabox( $feed_post ) {
 					<input type="checkbox" name="use_post_id_as_guid" value="1" <?php checked( $settings['use_post_id_as_guid'] ); ?> />
 				</td>
 			</tr>
+
+			<?php
+			// Only show this new option if the Republication Tracker Tool plugin is active.
+			if ( self::is_republication_tracker_plugin_active() ) :
+				?>
+				<tr>
+					<th>
+						<?php esc_html_e( 'Only include republishable posts', 'newspack-plugin' ); ?>
+						<p class="description"><?php echo esc_html_x( '(When toggled on, posts which have republication disabled will be excluded from the feed.)', 'help text for only republishable setting', 'newspack-plugin' ); ?></p>
```

Comment:
Since this is already styled as a help/description text, there's no need for parentheses:
```suggestion
						<p class="description"><?php echo esc_html_x( 'When toggled on, posts which have republication disabled will be excluded from the feed.', 'help text for only republishable setting', 'newspack-plugin' ); ?></p>
```